src/commands/threat-feed/cmd-threat-feed.ts

```diff
@@ -80,19 +110,16 @@ async function run(
     return
   }
 
-  const apiToken = getDefaultToken()
-  if (!apiToken) {
-    throw new AuthError(
-      'User must be authenticated to run this command. To log in, run the command `socket login` and enter your API key.'
-    )
-  }
-
   await getThreatFeed({
-    apiToken,
     direction: String(cli.flags['direction'] || 'desc'),
+    ecoSystem: String(cli.flags['ecoSystem'] || ''),
     filter: String(cli.flags['filter'] || 'mal'),
-    outputJson: Boolean(cli.flags['json']),
-    page: String(cli.flags['filter'] || '1'),
-    perPage: Number(cli.flags['per_page'] || 0)
+    outputKind: cli.flags['json']
+      ? 'json'
+      : cli.flags['markdown']
+        ? 'markdown'
+        : 'print',
+    page: String(cli.flags['page'] || '1'),
```

Comment:
```suggestion
    page: Number(cli.flags['page'] || '1'),
```
Should we make that a number? (requires tweaking other places too)

Reply:
I'm basically following the server api reference which expects a string here.

Going to keep it like this for now. We can revisit this (and all the other cases) if we change our mind.